internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -0,0 +1,259 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package quickstart
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/convert"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/randgen"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const quickstartTemplate = "Now you can connect to your Atlas cluster with: mongo -u %s -p %s %s\n"
+
+const (
+	replicaSet        = "REPLICASET"
+	mdbVersion        = "4.4"
+	shards            = 1
+	tier              = "M2"
+	tenant            = "TENANT"
+	members           = 3
+	zoneName          = "Zone 1"
+	accessListComment = "IP added with mongocli atlas quickstart"
+	atlasAdmin        = "atlasAdmin"
+	none              = "NONE"
+	passwordLength    = 12
+)
+
+// DefaultRegions represents the regions available for each cloud service provider
+var DefaultRegions = map[string][]string{
+	"AWS":   {"US_EAST_1", "US_WEST_2", "AP_SOUTH_1", "AP_EAST_2", "EU_WEST_1", "EU_CENTRAL_1", "ME_SOUTH_1", "AF_SOUTH_1"},
+	"GCP":   {"CENTRAL_US", "CANADA_CENTRAL", "WESTERN_EUROPE", "ASIA_SOUTH_EAST", "SOUTH_AFRICA_NORTH", "UAE_NORTH"},
+	"AZURE": {"US_EAST_2", "US_WEST", "EUROPE_NORTH"},
+}
+
+type Opts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	ClusterName      string
+	Provider         string
+	Region           string
+	IPAddress        string
+	DBUsername       string
+	DBUserPassword   string
+	connectionString string
+	store            store.AtlasClusterQuickStarter
+}
+
+func (opts *Opts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *Opts) Run() error {
+	if err := opts.askClusterFlags(); err != nil {
+		return err
+	}
+
+	if _, err := opts.store.CreateCluster(opts.newCluster()); err != nil {
+		return err
+	}
+
+	if err := opts.askDBUserAccessListFlags(); err != nil {
+		return err
+	}
+
+	if _, err := opts.store.CreateDatabaseUser(opts.newDatabaseUser()); err != nil {
+		return err
+	}
+
+	// Add IP to project’s IP access list
+	entry := opts.newProjectIPAccessList()
+	if _, err := opts.store.CreateProjectIPAccessList(entry); err != nil {
+		return err
+	}
+
+	fmt.Println("Creating your cluster...")
+	if er := opts.Watch(opts.watcher); er != nil {
+		return er
+	}
+
+	// Get cluster's connection string
+	cluster, err := opts.store.AtlasCluster(opts.ConfigProjectID(), opts.ClusterName)
+	if err != nil {
+		return err
+	}
+	opts.connectionString = cluster.SrvAddress
+
+	fmt.Printf(quickstartTemplate, opts.DBUsername, opts.DBUserPassword, opts.connectionString)
```

Comment:
[q] why `opts.connectionString`? any advantage over
```suggestion
	fmt.Printf(quickstartTemplate, opts.DBUsername, opts.DBUserPassword, cluster.SrvAddress)
```